src/test/java/com/segment/analytics/android/integrations/adjust/AdjustIntegrationTest.java

```diff
@@ -160,6 +160,22 @@
     verify(adjustInstance).trackEvent(event);
   }
 
+  @Test public void trackWithAnonymousId() throws Exception {
```

Comment:
Shoudl have a test case of `trackWithUserId` too I think.

Reply:
Added